libnetwork/sandbox_store.go

```diff
@@ -206,9 +205,17 @@ func (c *Controller) sandboxRestore(activeSandboxes map[string]interface{}) erro
 			sb.restoreResolvConfPath()
 			create = !sb.config.useDefaultSandBox
 		}
+
+		ctx := context.TODO()
```

Comment:
Looks like punching-through the context is a realistic option (as a follow-up); I see this function is called from `libnetwork.New`, which already has a context and spans set up.

(Not sure if context-cancelation is expected to be handled in this function though, or if it would require a `context.WithoutCancel()`.